build/single.html

```diff
@@ -6,21 +6,63 @@
     <link rel="stylesheet" type="text/css" href="style.css" />
     <link rel="stylesheet" type="text/css" href="styles/responsive.css" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,700" rel="stylesheet" />
+    <link rel="stylesheet" href="http://i.icomoon.io/public/48159919ac/test/style.css"></link>
+    <link rel="stylesheet" type="text/css" href="fonts/icomoon/icomoon.css" />
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
     <script type="text/javascript" src="scripts/interaction.js"></script>
 </head>
 <body>
+    <div class="banner">
+
+    </div>
     <div class="header">
         <div class="wrapper">
             <a class="logo" href="#">
                 <img src="img/logo.png" alt="" />
             </a>
+            <h2 class="trouble">Troubleshooting</h2>
         </div>
     </div>
 
     <div class="article full section content">
```

Comment:
you're not using the `full` class anywhere in the CSS, so go ahead and remove it